core/sqf/sqenvcom.sh

```diff
@@ -139,6 +139,10 @@ else
 fi
 export TRAF_HOME=$PWD
 
+# normal installed location, can be overridden in .trafodion
+export DCS_INSTALL_DIR=$TRAF_HOME/dcs-$TRAFODION_VER
+export REST_INSTALL_DIR=$TRAF_HOME/rest-$TRAFODION_VER
+
```

Comment:
@mkby TRAFODION_VER is defined very near the beginning of this same file.  This file (sqenvcom.sh) is the location that is modified when we roll to a new release number.